### PR TITLE
[Fixed JENKINS-31585] Make workflow editor (ACE) resizable.

### PR DESCRIPTION
**Pull request by tfennelly**
[JENKINS-31585](https://issues.jenkins-ci.org/browse/JENKINS-31585)

ACE Editor doesn't support drag resize so this PR uses jQuery UI and it's "resizable" function to accomplish it.

### CODE REVIEW

**Comment by tfennelly** [comment]
@reviewbybees

File: cps/src/main/js/workflow-editor.js [hunk 1]
```diff
@@ -57,6 +57,19 @@ jenkinsJSModules.import('ace-editor:ace-editor-122')
...
+            $wfEditor.wrap('<div class="jquery-ui-1"></div>');
+            $wfEditor.resizable({
+                handles: "s", // Only allow vertical resize off the bottom/south border
+                resize: function() {
+                    editor.resize();
+                }
+            });
+            // Make the bottom border a bit thicker as a visual cue.
+            // May not be enough for some people's taste.
+            $wfEditor.css('border-bottom-width', '0.4em');
```
Comment (amuniz) [inline cps/src/main/js/workflow-editor.js]: Could it be extracted to a CSS file?

**Comment by amuniz** [comment]
Nice Tom! :bee:

File: cps/src/main/js/workflow-editor.js (hunk 1 above)
Comment (tfennelly) [inline cps/src/main/js/workflow-editor.js]: We'd need to add css specific to this adjunct. We could do that, but I think I'd wait 'till we have more style.

Comment (recena) [inline cps/src/main/js/workflow-editor.js]: :+1: for waiting future changes.

**Comment by jglick** [comment]
Have no idea what I am looking at but :bee:

**Pull request merged**